Sources/Client/ShopClient.swift

```diff
@@ -73,8 +68,8 @@ public final class ShopClient: RemoteService {
   /**
    * Shop service.
    */
-  internal func service(_ apiKey: APIKey) -> Shop {
-    let svc = RPCServiceFactory<Shop>.factory(forService: Transport.config.services.shop)
+  internal func service(_ apiKey: APIKey) -> ShopService {
+    let svc = RPCServiceFactory<ShopService>.factory(forService: Transport.config.services.shop)
```

Comment:
some basic consistency changes here - we have `MenuService`, we should also have `ShopService`

Reply:
Perfect is the enemy of done.